frontend/src/components/pages/connect/Connector.Details.tsx

```diff
@@ -204,7 +204,7 @@ const KafkaConnectorMain = observer(
                   <Flex m={4} mb={6}>
                     <Tooltip
                       placement="top"
-                      isDisabled={canEdit !== true}
+                      isDisabled={canEdit === true}
```

Comment:
Jesus that's indeed a little brain teaser..

Reply:
hehe agree